audit/src/main.ts

```diff
@@ -18,16 +27,16 @@ const setupEventBus = async (freshEventBus: EventBus) => {
 
   eventBus.subscribe<ServiceStartedPayload>(
     serviceStartedIdentifier,
-    ServiceStartedHandler(undefined)
+    ServiceStartedHandler(auditController)
   );
 
   // publish "ServiceStarted"
   const event: Event<ServiceStartedPayload> = {
     id: "some-wevent-id",
     created: new Date(),
     payload: {
-      type: "support",
-      name: "audit"
+      type: "support/audit",
```

Comment:
Should type be stored as a variable somewhere? or is this the only place it will ever need to be declared?

Reply:
I think we will review this in another PR